src/components/Notes/Notes.tsx

```diff
@@ -106,14 +106,17 @@ export const Notes = () => {
 					<div className={classes.buttons}>
 						<SearchBar disabled={!viewCondition} title='notes' />
 						<FilterPopup disabled={!viewCondition} />
+						{notes.length > 2 ? (
```

Comment:
Please use && instead of a ternary
```suggestion
						{notes.length > 2 && (
```